io.github.realmazharhussain.GdmSettings.yaml

```diff
@@ -2,7 +2,7 @@ app-id: io.github.realmazharhussain.GdmSettings
 command: gdm-settings
 sdk: org.gnome.Sdk
 runtime: org.gnome.Platform
-runtime-version: "46"
+runtime-version: "master"
```

Comment:
GDM Settings is not a GNOME offering. GNOME apps need to use the nightly runtime because they are supposed to be developed and released alongside it. However, third party apps should depend on a versioned runtime.

Until GNOME 47 beta (or maybe even stable) is released, GDM Settings' main branch will use GNOME 46 runtime.

Reply:
I used it to build the project, it was development intended.